spec/send_sms_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'send_sms'
+
+describe Send_sms do
+
+  let(:phonenumber){double :phonenumber}
```

Comment:
Space missing to the left of {.<br>Space missing inside {.<br>Space missing inside }.